From a38be95e3d7ed9d5060876e0b7ae3a1bb9bb32dd Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 25 Jun 2018 21:23:18 -0600
Subject: [PATCH] hudson: klte* wants to dunk its Oreos into some milk too

* Well, most of them. There's still that demented sprsports thing...

Change-Id: I03746b4f64d28b43198ca0932bac24e66aebf711
---
 lineage-build-targets | 15 ++++++++-------
 updater/devices.json  |  1 +
 2 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/lineage-build-targets b/lineage-build-targets
index eaab227..b3c8750 100644
--- a/lineage-build-targets
+++ b/lineage-build-targets
@@ -19,6 +19,14 @@ gts210vewifi userdebug lineage-15.1 W 1
 gts28vewifi userdebug lineage-15.1 W 1
 
 kccat6 userdebug lineage-15.1 W 2
+klte userdebug lineage-15.1 W 2
+klteactivexx userdebug lineage-15.1 W 2
+kltechn userdebug lineage-15.1 W 2
+kltechnduo userdebug lineage-15.1 W 2
+klteduos userdebug lineage-15.1 W 2
+kltedv userdebug lineage-15.1 W 2
+kltekdi userdebug lineage-15.1 W 2
+kltekor userdebug lineage-15.1 W 2
 lentislte userdebug lineage-15.1 W 2
 lithium userdebug lineage-15.1 W 2
 mako userdebug lineage-15.1 W 2
@@ -110,13 +118,6 @@ kingdom userdebug cm-14.1 W 5
 kipper userdebug cm-14.1 W 5
 kiwi userdebug cm-14.1 W 5
 klimtwifi userdebug cm-14.1 W 5
-klte userdebug cm-14.1 W 5
-kltechn userdebug cm-14.1 W 5
-kltechnduo userdebug cm-14.1 W 5
-klteduos userdebug cm-14.1 W 5
-kltedv userdebug cm-14.1 W 5
-kltekdi userdebug cm-14.1 W 5
-kltekor userdebug cm-14.1 W 5
 kltesprsports userdebug cm-14.1 W 5
 kuntao userdebug cm-14.1 W 5
 land userdebug cm-14.1 W 5
diff --git a/updater/devices.json b/updater/devices.json
index 9b6c32e..d103337 100644
--- a/updater/devices.json
+++ b/updater/devices.json
@@ -103,6 +103,7 @@
   {  "model": "kiwi", "oem": "Huawei", "name": "Honor 5X"},
   {  "model": "klimtwifi", "oem": "Samsung", "name": "Galaxy Tab S 8.4 (Wi-Fi)"},
   {  "model": "klte", "oem": "Samsung", "name": "Galaxy S5 LTE (G900AZ/F/M/R4/R7/T/V/W8,S902L)"},
+  {  "model": "klteactivexx", "oem": "Samsung", "name": "Galaxy S5 Active (SM-G870F)"},
   {  "model": "kltechn", "oem": "Samsung", "name": "Galaxy S5 LTE (G9006V/8V)"},
   {  "model": "kltechnduo", "oem": "Samsung", "name": "Galaxy S5 LTE Duos (G9006W/8W)"},
   {  "model": "klteduos", "oem": "Samsung", "name": "Galaxy S5 LTE Duos (G900FD/MD)"},
-- 
2.17.1

